projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -302,7 +312,11 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
     }
 
     const validateTimer = timer(10).subscribe(() => {
-      this.workflowStepAllowed(this.inputFields ? this.inputFields.toArray() : undefined);
+      if (this.inputFields) {
+        this.workflowStepAllowed(this.inputFields.toArray());
+      } else {
+        this.workflowStepAllowed(undefined);
+      }
```

Comment:
For consistency, perhaps do this (lines 315-319) the same way as you did it below (lines 331-335)? I think that way is better, don't you?

Reply:
Yes